From 3745f65c84fe600cc2fd1001f0d51c8d06db162a Mon Sep 17 00:00:00 2001
From: kxxt <rsworktech@outlook.com>
Date: Fri, 2 Feb 2024 16:36:25 +0000
Subject: [PATCH 1/8] disable cpuinfo build for riscv64

Not supported for riscv64

Change-Id: I23e7a1ec0157b73e4969b582a8cba8e2d12d8c10
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5256319
Auto-Submit: Levi Zim <rsworktech@outlook.com>
Reviewed-by: Robert Ogden <robertogden@chromium.org>
Reviewed-by: Michael Crouse <mcrouse@chromium.org>
Commit-Queue: Michael Crouse <mcrouse@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1255602}
---
 AUTHORS                         | 1 +
 third_party/cpuinfo/cpuinfo.gni | 1 +
 2 files changed, 2 insertions(+)

diff --git a/AUTHORS b/AUTHORS
index db4cd10cbe50f..4b6849dd8cac4 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -804,6 +804,7 @@ Lenny Khazan <lenny.khazan@gmail.com>
 Leo Wolf <jclw@ymail.com>
 Leon Han <leon.han@intel.com>
 Leung Wing Chung <lwchkg@gmail.com>
+Levi Zim <rsworktech@outlook.com>
 Li Yanbo <liyanbo.monster@bytedance.com>
 Li Yin <li.yin@intel.com>
 Lian Ruilong <lianrl@dingdao.com>
diff --git a/third_party/cpuinfo/cpuinfo.gni b/third_party/cpuinfo/cpuinfo.gni
index be4e1554f540b..f7a27ee82601b 100644
--- a/third_party/cpuinfo/cpuinfo.gni
+++ b/third_party/cpuinfo/cpuinfo.gni
@@ -6,6 +6,7 @@ use_cpuinfo =
     # cpuinfo is not supported on these chips.
     current_cpu != "ppc64" && current_cpu != "ppc64le" &&
     current_cpu != "s390" && current_cpu != "s390x" &&
+    current_cpu != "riscv64" &&
     # cpuinfo is not supported on fuchsia.
     !is_fuchsia &&
     # There's a few arm architectures that are not supported by cpuinfo,
-- 
2.43.0

